rules/prefer_to_have_length.js

```diff
@@ -1,46 +1,44 @@
 'use strict';
+const {
```

Comment:
Destructure doesn't work in node 4. We can revisit dropping node 4 later, but for now can you make this work there?